dashboards/redis-otel/redis-otel.json

```diff
@@ -0,0 +1,612 @@
+{
+  "name": "Redis OTel",
+  "description": null,
+  "pages": [
+    {
+      "name": "Redis OTel",
+      "description": null,
+      "widgets": [
+        {
+          "title": "",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 3,
+            "height": 4
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "![Redis logo](https://upload.wikimedia.org/wikipedia/commons/thumb/6/64/Logo-redis.svg/300px-Logo-redis.svg.png?20210820223643)\n\n### Overview\n[Redis](https://redis.io/) is an in-memory data structure store, used as a database, cache, and message broker. Redis allows you to achieve high performance by keeping your data in memory, but also offers durability options. Its support for a variety of data structures, such as strings, hashes, sets, and lists, has made Redis a versatile tool suitable for a vast number of use cases.\n\n- - -\n### About this Dashboard\n\n- For information on the prerequisites for this dashboard, please visit the documentation [here](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/redis/redis-integration/).\n- For more information on how to send Redis telemetry data to New Relic using OTEL, visit the documentation [here](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-redis).\n"
```

Comment:
This link does not seem to exist on our documentation site.

Furthermore, we are now trying to avoid linking to assets (e.g. images) hosted on non-newrelic servers due to security concerns. I _think_ wikimedia is fine, but if we have a newrelic-hosted image, that would be preferable.

Reply:
Hi @zstix 
How can we upload images to a newrelic host?

Reply:
We have one hosted on our .com site: https://newrelic.com/sites/default/files/styles/medium/public/quickstarts/images/icons/redis--logo.png

I wouldn't block the PR for this (wikimedia is fairly trustworthy), but we're trying to be more careful about external assets.